tt_eager/tensor/borrowed_buffer.hpp

```diff
@@ -32,7 +32,7 @@ struct Buffer {
 
     inline void* data() noexcept { return static_cast<void*>(this->data_ptr_); }
     inline const void* data() const noexcept { return static_cast<void*>(this->data_ptr_); }
-
+    inline std::shared_ptr<std::vector<T>> copy_to_vec() const noexcept { return std::make_shared<std::vector<T>>(std::vector<T>(this->begin(), this->end())); }
```

Comment:
I see this is only really used to created an `OwnedBuffer`.. Can we use the existing `owned_buffer::create(..)` function to do that instead?

Reply:
Added to latest version of PR.